src/sentry/static/sentry/app/views/app.jsx

```diff
@@ -222,7 +222,7 @@ const App = createReactClass({
   render() {
     if (this.state.loading) {
       return (
-        <LoadingIndicator triangle={true}>
+        <LoadingIndicator>
```

Comment:
I think the swirling triangle was appealing. But maybe we shouldn't use the obvious sentry stuff?

Reply:
I agree that it looks cool, but it's their logo.

We could easily make a swirling icon of our own later though!